src/compression.rs

```diff
@@ -36,33 +38,19 @@ use crate::{
     Result,
 };
 
-/// Contains a fixed list of common text-based MIME types in order to apply compression.
-pub const TEXT_MIME_TYPES: [&str; 24] = [
```

Comment:
Unfortunately, this was public, so this is a breaking change.

Reply:
Let's keep the comment open for mentioning in the release.